mender-convert-functions.sh

```diff
@@ -396,10 +396,11 @@ unmount_partitions() {
 create_mender_disk() {
   local lfile=$1
   local lsize=$2
-  local bs=$(( 1024*1024 ))
-  local count=$(( ${lsize} / ${bs} ))
 
-  dd if=/dev/zero of=${lfile} bs=${bs} count=${count}>> "$build_log" 2>&1
+  log "\tGenerating a blank image..."
```

Comment:
I would prefer modifying `Creating Mender disk image:` comment in _mender-convert_ script to `Creating blank Mender disk image: ` instead of adding a new log here.

Reply:
Ack